rq/worker.py

```diff
@@ -175,7 +176,8 @@ def __init__(self, queues, name: t.Optional[str] = None, default_result_ttl=DEFA
                  queue_class=None, log_job_description: bool = True,
                  job_monitoring_interval=DEFAULT_JOB_MONITORING_INTERVAL,
                  disable_default_exception_handler: bool = False,
-                 prepare_for_work: bool = True, serializer=None):  # noqa
+                 prepare_for_work: bool = True, serializer=None,
+                 workhorse_terminated_handler=None):  # noqa
```

Comment:
Mind updating the docs for this new argument?

Reply:
@selwin sure, where is it documented? I didn't find any place documenting arguments to the constructor

Reply:
Thanks for updating this PR. Mind adding some documentation for this handler in the exceptions section?

This is how it looks like on the web. https://python-rq.org/docs/exceptions/#custom-exception-handlers